channeldb/graph.go

```diff
@@ -56,19 +56,25 @@ var (
 	// as an adjacency list, which in conjunction with a range scan, can be
 	// used to iterate over all the _outgoing_ edges for a particular node.
 	// Key in the bucket use a prefix scheme which leads with the node's
-	// public key and sends with the compact edge ID. For each edgeID,
+	// public key and sends with the compact edge ID. For each chanID,
 	// there will be two entries within the bucket, as the graph is
 	// directed: nodes may have different policies w.r.t to fees for their
 	// respective directions.
 	//
-	// maps: pubKey || edgeID -> edge policy for node
+	// maps: pubKey || chanID -> channel edge policy for node
```

Comment:
I say we deprecate the value here, and instead use it as a set of edges. Then we can retrieve both incoming and outgoing policies the same way.

Reply:
What scheme do you exactly propose here? pubKey -> subbucket of edges?

I thought we extensively discussed on slack the pros and cons of both the unknown value and the subbucket approach and in the end decided to create an unknown value without further db changes. Therefore this is what I implemented. Do you currently think otherwise?

Reply:
I propose `maps: pubKey || edgeID -> "unknown value"`, where "unknown value" is just a dummy value that indicates this edge exists.

This is in essence what you are doing already, I'm just suggesting we shouldn't decode and use the dummy value for anything.